packages/dashmate/docker/entrypoint.sh

```diff
@@ -5,30 +5,37 @@ DOCKER_GROUP_ID=$(stat -c %g /var/run/docker.sock)
 USERNAME=dashmate
 GROUP=docker
 
-# check if user with our uid exists in the system
```

Comment:
Had you changed only indentation? We have a PR where guys are fixing logic there https://github.com/dashpay/platform/pull/1072

Reply:
Agree!